doc/orchestrator.rst

```diff
@@ -63,108 +326,207 @@ on that compute host.
 
 |colo-orc|
 
-This deployment is designed for highly performant online inference scenarios where
+Colocated deployment is designed for highly performant online inference scenarios where
 a distributed process (likely MPI processes) are performing inference with
-data local to each process.
+data local to each process. Data produced by these processes and stored in the colocated database
+can be transferred via a SmartRedis client to a standard database to become available for consumption
+by other applications. A tradeoff of colocated deployment is the ability to scale to a large workload.
+Colocated deployment rather benefits small/medium simulations with low latency requirements.
+By hosting the database and simulation on the same compute node, communication time is reduced which
+contributes to quicker processing speeds.
 
 This method is deemed ``locality based inference`` since data is local to each
 process and the ``Orchestrator`` is deployed locally on each compute host where
 the distributed application is running.
 
+-------
+Example
+-------
+In the following example, we provide a demonstration on automating the deployment of
+a colocated Orchestrator using SmartSim from within a Python driver script. Once the colocated database is launched,
+we demonstrate connecting a client to the database from within the application script to transmit and poll data.
 
-To create a colocated model, first, create a ``Model`` instance and then call
-the ``Model.colocate_db_tcp`` or ``Model.colocate_db_uds`` function.
+The example is comprised of two script files:
+
+- The Application Script
+   The example application script is a Python file that contains
+   instructions to create and connect a SmartRedis
+   client to the colocated Orchestrator.
+- The Experiment Driver Script
+   The experiment driver script launches and manages
+   the example entities with the ``Experiment`` API.
+   In the driver script, we use the ``Experiment``
+   to create and launch a colocated ``Model``.
 
-.. currentmodule:: smartsim.entity.model
+The Application Script
+======================
+To begin writing the application script, provide the imports:
 
-.. automethod:: Model.colocate_db_tcp
-    :noindex:
+.. code-block:: python
 
-.. automethod:: Model.colocate_db_uds
-    :noindex:
+  from smartredis import ConfigOptions, Client, log_data
+  from smartredis import *
+  import numpy as np
 
-Here is an example of creating a simple model that is colocated with an
-``Orchestrator`` deployment using Unix Domain Sockets
+Initialize the Clients
+----------------------
+To establish a connection with the colocated database,
+initialize a new SmartRedis client and specify `cluster=False`
+since our database is single-sharded:
```

Comment:
our database is single-sharded: -> colocated deployments are always single-sharded: